arches/app/models/graph.py

```diff
@@ -563,19 +563,18 @@ def delete(self):
                 )
             )
 
-    def delete_instances(self, verbose=False):
+    def delete_instances(self, userid=None, verbose=False):
```

Comment:
I make a habit of adding new optional arguments to the end of signatures to improve backwards compatibility.

Reply:
If they're both kwargs does order still matter?

Reply:
Yeah, unless they're keyword-only with `*,` to consume the positional args, which we don't have here. I floated starting to do this and got feedback that we don't do it anywhere else in Arches so not to bother with it.

Reply:
in my humblest of opinions on kwarg order, userid makes more sense as an earlier argument since it's more important. I'm not married to it but would be great to merge this sooner rather than later

Reply:
Finally got around to the forum post: https://community.archesproject.org/t/preferring-keyword-only-parameters-when-adding-functionality-to-existing-functions/2538